test/python/primitives/test_backend_estimator.py

```diff
@@ -322,6 +338,29 @@ def test_no_max_circuits(self):
             self.assertEqual(len(result.metadata), k)
             np.testing.assert_allclose(result.values, target.values, rtol=0.2, atol=0.2)
 
+    def test_bound_pass_manager(self):
+        """Test bound pass manager."""
+
+        bound_counter = LogPass("bound_pass_manager")
+        bound_pass = PassManager(bound_counter)
+
+        estimator = BackendEstimator(backend=FakeNairobi(), bound_pass_manager=bound_pass)
+
+        qc = QuantumCircuit(2)
+        op = SparsePauliOp.from_list([("II", 1)])
+
+        with self.subTest("Test single circuit"):
+            with self.assertLogs(logger, level="INFO") as cm:
+                _ = estimator.run(qc, op).result()
+            expected = ["bound_pass_manager"]
+            self.assertEqual([record.message for record in cm.records], expected)
```

Comment:
Is it better to access `cm` within the `with` block?

Reply:
I am not sure of what the implications are, but just in case I moved to to inside the `with` block :)

Reply:
I'm wondering whether `cm` is valid after the with-statement because `cm.__exit__` is called. If you are sure that `cm` is valid after the with-statement, I'm fine with the original code.